lib/ansible/modules/cloud/vmware/vmware_guest.py

```diff
@@ -201,6 +201,12 @@
     - "vmware-tools needs to be installed on the given virtual machine in order to work with this parameter."
     default: 'no'
     type: bool
+  wait_for_customization:
+    description:
+    - Wait until vCenter detects all guest customizations as successfully completed.
```

Comment:
Add `When enabled, the VM will automatically be powered on.`

Reply:
done